conf/serialization.conf

```diff
@@ -144,6 +146,7 @@ akka.actor {
     "models.rpc.PongMessage" = kryo
     "models.rpc.ResponseMessage" = kryo
     "models.rpc.RequestMessage" = kryo
+    "models.rpc.DSAError" = kryo
```

Comment:
I believe this is redundant. That error we saw in the log says about Kryo mapping only, means it is already bind.